CONTRIBUTING.md

```diff
@@ -130,20 +109,50 @@ If your PR has only one commit, the PR title can be the same as the commit title
 suggest this). If the PR has multiple commits, do not accept the title GitHub suggests. Either use
 the title of the most relevant commit, or write your own title.
 
-## Document Your Changes
+## Contributor Guidelines
 
-If you have added or modified any user-facing functionality, such as CLI commands or their output,
-then the pull request must include appropriate updates to documentation.
+Do not merge directly into the `main` branch. It is protected for a reason.
 
-It is the submitter's responsibility to make the changes, and the reviewer's responsibility to make
-sure they are not merging changes that do not have the needed updates to documentation.
+### Reviewing Pull Requests
 
-## Running Unit Tests
+Each pull request requires at least *one* review and approval from an active
+contributor. The primary function of reviews is to ensure ensure quality. However,
+we want pull requests to be viewed quickly to avoid code contributors being blocked.
 
-Before submitting your change, you should test it. You can run unit tests with tox:
+We ask that reviewers perform a basic sanity check on all code and documentation
+contributions. Consider the following aspects when performing a review:
+
+- Does this make sense?
+- Does this code perform what the code contributor has described in the commit message?
+- Have the CI tests run and passed?
+- Does this code require documentation? If so, does it have documentation
+  in the commit?
+- Are the comments (if there are any) are clear and useful?
```

Comment:
extra `are` before `clear`.

Reply:
This isn't new text; It's refactored from before. Good catch nonetheless so I've added another commit.